bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/SuuToChiChi_FullyHadronic/SuuToChiChi_FullyHadronic_MSuu8TeV_MChi3TeV/SuuToChiChi_FullyHadronic_MSuu8TeV_MChi3TeV_proc_card.dat

```diff
@@ -0,0 +1,73 @@
+#************************************************************
+#*                     MadGraph5_aMC@NLO                    *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.7.3                 2020-06-21         *
+#*                                                          *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph5_aMC@NLO         *
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+set default_unset_couplings 99
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+import model diquarkVLQ2020_UFO
+generate p p > suu >  W+ b H t, W+ > j j, (t > W+ b, W+ > j j),(H > b \
+b~)
+add process p p > suu >  W+ b H t, W+ > j j, (t > W+ b, W+ > j j),(H >\
+ Z Z, Z > j j , Z > j j)
+add process p p > suu >  W+ b H t, W+ > j j, (t > W+ b, W+ > j j),(H >\
+ W+ W-, W+ > j j, W- > j j)
+add process p p > suu > H t Z t, (t > W+ b, W+ > j j), Z > j j,(t > W+\
+ b, W+ > j j),(H > b b~)
+add process p p > suu > H t Z t, (t > W+ b, W+ > j j), Z > j j,(t > W+\
+ b, W+ > j j),(H > Z Z, Z > j j , Z > j j)
+add process p p > suu > H t Z t, (H > W+ W-, W+ > j j, W- > j j), (t >\
+ W+ b, W+ > j j), Z > j j,(t > W+ b, W+ > j j)
+add process p p > suu > H t H t,(H > b b~),(H > b b~)
+add process p p > suu > H t H t,(H > b b~),(H > Z Z, Z > j j , Z > j j\
+)
```

Comment:
there should be overlap between the following two lines:
p p > suu > H t H t, (H > b b~), (H > Z Z, Z > ...)
p p > suu > H t H t, (H > Z Z, Z > ...), (H > b b~)
one of them should be enough. 
e.g., if you try " p p > z z" with default MG setup, the cross section is around 9.335pb, for process " p p > z z, z > e+ e-, z > b b~" with "false = cut_decay", the cross section is around 0.092, which approximate 0.034*0.156*9.335*2. could you please confirm it

Reply:
Understood, I have gone through the processes that I add and made sure that everything is being decayed as I want and that there are no duplicate processes